build.gradle

```diff
@@ -1226,15 +1226,27 @@ project(':generator') {
 }
 
 project(':clients') {
+  apply plugin: 'com.github.johnrengelman.shadow'
+
   archivesBaseName = "kafka-clients"
 
+  shadowJar {
+    zip64 = true
+    relocate('io.opentelemetry', 'shadow.io.opentelemetry')
```

Comment:
typically we relocate packages to something in our own namespace, e.g. `io.confluent.shaded`.
If we plan to include this as part of open source clients, we'd probably want to put this under `org.apache.kafka.shaded`